ui/src/RoomManage.tsx

```diff
@@ -72,29 +69,20 @@ const CreateRoom = ({room, config}: Pick<UseRoom, 'room'> & {config: UIConfig})
                     }
                     label="Close Room after you leave"
                 />
-                <FormControl margin="dense">
-                    <InputLabel>NAT Traversal via</InputLabel>
-                    <Select
-                        fullWidth
-                        value={mode}
-                        onChange={(x) => setMode(x.target.value as RoomMode)}
-                        endAdornment={
+                <TextField
```

Comment:
Can we use just text here, with the text field it still displays the "edit" cursor. In material ui the <Typography component is used for that.

Reply:
Ok thanks for the hint 🙂 How does this look to you?

![image](https://user-images.githubusercontent.com/1177900/119378952-21fdf900-bcbf-11eb-8b1d-515ab316a336.png)

Reply:
I think a link instead of the help button would be better (couldn't align the icon with the test (: )
```
                <Typography>
                    Nat Traversal via:{' '}
                    <Link href="https://screego.net/#/nat-traversal">{mode.toUpperCase()}</Link>
                </Typography>
```

Reply:
Looks good to me! What do you think about `target="_blank"`, to not loose the screensharing page while navigating to read the docs? I added but can remove.

Reply:
I'd also add some padding between the `Typography` and button under it, but not sure how to properly do it, just add some style with some hardcoded value like `5px`?

Reply:
You can use Box (from "@material-ui/core" with paddingBottom, 1 = 8px and 0.5 looks about right.
```typescript
<Box paddingBottom={0.5}>
    <Typography>
        Nat Traversal via:{' '}
        <Link
            href="https://screego.net/#/nat-traversal"
            target="_blank"
            rel="noreferrer">
            {mode.toUpperCase()}
        </Link>
    </Typography>
</Box>
```

Reply:
Thanks! For my eye `2` looks better because it achieves similar padding like between each field (see below), what do you think? Not feeling strongly about it 😉 

![image](https://user-images.githubusercontent.com/1177900/119385483-78bb0100-bcc6-11eb-9c0a-4be0865e4fa6.png)

Reply:
I feel like 0.5 are better
![image](https://user-images.githubusercontent.com/14895212/119385902-5a4d0980-bcb6-11eb-8d5a-9feceb9ccb44.png)

Reply:
Oki 🙂